web/web.go

```diff
@@ -379,6 +379,12 @@ func (h *Handler) targets(w http.ResponseWriter, r *http.Request) {
 		tps[job] = append(tps[job], t)
 	}
 
+	for _, targets := range tps {
+		sort.Slice(targets, func(i, j int) bool {
```

Comment:
Using sort.Slice makes go1.8 a requirement now. If that's desired, we'll need to update the build+ci infrastructure first.